my-sms/app/controllers/institutes_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: false
+
+class InstitutesController < ApplicationController
+  decorates_assigned :institute
+
+  before_filter :load_institute, except: %i[index]
+  before_render :flash_errors, only: %i[edit create update]
+
+  # GET /institutes
+  # GET /institutes.json
```

Comment:
Scaffold comments make it messy .. please remove

Reply:
Fixed in a3abd116c79511f50e413bc05c5b67319668f038

FYI I wasn't using scaffold generators for the institutes controller, I just duplicated them to match the original students controller.